.gitignore

```diff
@@ -9,3 +9,5 @@ release.properties
 
 generated-sources/
 *.log
+
+.idea/
```

Comment:
This directory is environment specific, please move it to your global `.gitignore` in your user's home dir.

Reply:
Nah, it's fine here. Let's make life simplers for all participating devs, not harder.

Reply:
It's not harder @gunnarmorling, Sorry, I disagree! That's just how you usually do it, based on your environment you put your IDE specific entries into your user's `.gitignore` file in the home dir. We should not put all temp directories for all possible IDEs into the project's `.gitignore`. DRY!